### DISCUSSION TITLE
asus ac86u 原版梅林固件安装失败

### DISCUSSION THREAD

**Discussion by voq** [messense/aliyundrive-webdav#527] [Q&A] [answered]
### 问题描述

错误信息见日志

### 重现步骤

opkg install aliyundrive-webdav_1.8.4-1_aarch64_generic.ipk

### 版本

1.8.4-1

### 运行平台

ac86u

### refresh token 获取方式

无

### 日志

> a@RT-AC86U-36D8:/tmp/home/root# opkg install aliyundrive-webdav_1.8.4-1_aarch64_generic.ipk
> Unknown package 'aliyundrive-webdav'.
> Collected errors:
>  * pkg_hash_fetch_best_installation_candidate: Packages for aliyundrive-webdav found, but incompatible with the architectures configured
>  * opkg_install_cmd: Cannot install package aliyundrive-webdav.

**Comment by messense** [comment, marked as answer]
不支持。
